README.md

```diff
@@ -197,6 +197,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Check out the [Mattermost website](https://mattermost.com/blog/hacktoberfest-2021/) for more details
 - **Notes**: Check out the [Mattermost handbook for Hacktoberfest](https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/hacktoberfest2021) and their [community colab server](https://docs.mattermost.com/guides/community-chat.html)
 
+#### **MayaData & OpenEBS**
+
+- **Swag**: T-shirts ,Oculus VR Headset
+- **Requirements**:
+  - 1 or more pull requests to any eligible repository : special MayaData swags
+  - best pull request : Oculus VR Headset.
+- **How to sign up**: Check out this [blog](https://blog.mayadata.io/celebrate-hacktoberfest-2021-support-open-source-with-mayadata-openebs) for more details
+- **Notes**: Check out the [MayaData is celebrating Hacktoberfest 2021 with openEBS](https://mayadata.io/hacktoberfest-2021) to start contributing.
```

Comment:
Can you please also add the **Issues** bullet point too, as MayaData explicitly links [to this page](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Aopenebs+archived%3Afalse+label%3A%22help+wanted%22+)? Be sure to add that to the sections below as well. 👍